pkgs/development/libraries/fmt/default.nix

```diff
@@ -9,6 +9,10 @@ stdenv.mkDerivation rec {
     rev = "${version}";
     sha256 = "1cd8yq8va457iir1hlf17ksx11fx2hlb8i4jml8gj1875pizm0pk";
   };
+  patches = [
+    # https://github.com/fmtlib/fmt/pull/916
```

Comment:
Might use something like the following then:


```nix
patches = [
  (fetchPatch {
    name = "0001-add-fmt";
    url = "https://github.com/fmtlib/fmt/pull/916.patch";
    sha256 = "xxxxxxxxxxxxxxxxxxxxxxxxx";
  })
];

Reply:
Instead of the pull request please create a dedicated branch in this case, so the patch never changes, if upstreams wants you to change the pull request.